### DISCUSSION TITLE
no such column: stripe_id

### DISCUSSION THREAD

**Discussion by devhoussam1998** [maartenpaauw/filament-cashier-billing-provider#16] [Q&A] [answered]
when I visit the `/billing` I get this error: **no such column: stripe_id**

<img width="686" alt="Screenshot 2024-05-08 at 17 41 39" src="https://github.com/maartenpaauw/filament-cashier-billing-provider/assets/121255432/a289540a-6bb1-4e4f-a631-06fa552c2aa9">

**Comment by maartenpaauw** [comment, marked as answer]
Seems like you didn’t complete the Laravel Cashier setup. You should already have added the required columns to your teams table (using migrations) before you are able to use this plugin.
